### PR TITLE
Adapt showroom to new design

### PR DESCRIPTION
**Pull request by xaviervia**
This should be just a cosmetic change borne of the latest design update for the Showroom.

screen shot 2016-10-04 at 10 30 30
screen shot 2016-10-04 at 10 30 22
screen shot 2016-10-04 at 10 30 12

### CODE REVIEW

File: ProgressBar/example.jsx [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
-    'Custom inline styles': [
-      <ProgressBar percentage={50} style={{height: '20px', width: '70%'}} />
-    ]
-  }]
+    type: LIVE,
+
+    examples: {
+      Regular: <ProgressBar percentage={50} />,
+
+      Blue: <ProgressBar percentage={33} fillColor={"blue"} />,
```
Comment (JoakimLofgren) [inline ProgressBar/example.jsx]: You don't need the curlys for the values right?

File: Selector/example.jsx [hunk 2]
```diff
@@ -1,43 +1,61 @@
...
+          name='selector'
+          data={optionsData}
+        />
+      }
+    },
+
+    {
+      title: 'Direct Selector',
+      require: `import * as Selector from '@klarna/ui/Selector'
+    import UncontrolledSelector from '@klarna/ui/uncontrolled/Selector'`,
```
Comment (JoakimLofgren) [inline Selector/example.jsx]: No `UncontrolledSelector.Direct` example ?

File: Showroom/Footer/index.jsx [hunk 3]
```diff
@@ -1,37 +1,40 @@
 import React from 'react'
-import * as UI from '../../'
+import * as Button from '../../Button'
+import K from '../K'
+import Link from '../../Link'
+import * as Paragraph from '../../Paragraph'
+import * as Title from '../../Title'
+import {version} from '../../package.json'
```
Comment (JoakimLofgren) [inline Showroom/Footer/index.jsx]: Can't you instead add a [DefinePlugin](https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin) which exposes `VERSION` and maybe even `GIT_HASH` ?

File: Showroom/Footer/index.jsx [hunk 4]
```diff
@@ -1,37 +1,40 @@
...
-            Copyright © 2015-2016 Klarna AB. Version 1.0.0.
-          </UI.Paragraph.Legal>
-        </div>
+        <Paragraph.Legal
+          className={styles['footer__legal']}>
+          Copyright © 2015-2016 <Link
+            target='_blank'
+            href='https://klarna.com'>Klarna AB</Link>. Version <Link
+              target='_blank'
+              href={`https://github.com/klarna/ui/releases/tag/v${version}`}>{version}</Link>
```
Comment (JoakimLofgren) [inline Showroom/Footer/index.jsx]: Only in the footer? Maybe add it to the Getting Started page as well?

File: Showroom/Footer/index.jsx [hunk 5]
```diff
@@ -1,37 +1,40 @@
...
-        className={styles.getInTouch}
-        onClick={() => window.open('https://github.com/klarna/ui/issues')}>
-        Get in touch
-      </UI.Button.Secondary>
-    </footer>
-  )
+    <Button.Primary
+      size='small'
+      className={styles['footer__get-in-touch']}
+      onClick={() => window.open('https://github.com/klarna/ui/issues')}>
```
Comment (JoakimLofgren) [inline Showroom/Footer/index.jsx]: This kinda breaks the whole "copy link", any reason why not just style a Link differently ?
Or a Button-as-a-Link component maybe?

Firefox "blocks popups" when I middle click the button as well.

File: Showroom/K/styles.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  transition: fill .2s ease;
+
+  &.hoverable {
+    &:hover {
+      fill: map-get($colors, klarna-blue-hover);
+    }
+  }
+}
+
+// .logo {
```
Comment (JoakimLofgren) [inline Showroom/K/styles.scss]: Remove ?

File: Showroom/Main/index.scss [hunk 7]
```diff
@@ -4,6 +4,7 @@
 @import '../../css/fira-code/fira_code.css';
 
 body {
+  background-color: #f5f5f7;
```
Comment (JoakimLofgren) [inline Showroom/Main/index.scss]: No variable for this one?

File: Showroom/Page/Example/styles.scss [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+  .page__example__heading {
+    margin-bottom: 10px;
+    margin-left: 0;
+    margin-top: 35px;
+    width: 100%;
+  }
+
+  .page__example__live {
+    float: static;
```
Comment (JoakimLofgren) [inline Showroom/Page/Example/styles.scss]: `float: static`, supposed to be `position` instead ?

File: Showroom/Page/Variation/index.jsx [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+  navigation: `${baseClass}__navigation`,
+  navigationComponent: `${baseClass}__navigation__component`,
+  navigationReactCode: `${baseClass}__navigation__react--code`,
+  require: `${baseClass}__require`
+}
+
+export default function Variation ({type, ...props}) {
+  const classNames = classNamesBind.bind(styles)
+
+  switch (type) {
```
Comment (JoakimLofgren) [inline Showroom/Page/Variation/index.jsx]: Some cases seems kinda long, maybe split each case out to a function?

File: Showroom/Sidebar/styles.scss [hunk 10]
```diff
@@ -1,8 +1,11 @@
+@import '../../css/settings';
 @import '../settings.scss';
+@import '../../css/mixins/index';
 
 .sidebar {
   -webkit-font-smoothing: antialiased;
```
Comment (JoakimLofgren) [inline Showroom/Sidebar/styles.scss]: I've seen this a few times now I think.
Wouldn't this be more of a base style `html,body` rule than on certain elements only ?

File: Showroom/Sidebar/styles.scss [hunk 11]
```diff
@@ -14,27 +17,61 @@
...
-  padding-left: 20px;
+  margin-left: ($grid * 2);
+}
+
+.menuItemSelected {
+  color: map-get($colors, black);
+  font-weight: map-get($font-weights, semi-bold);
+}
+
+// .menuItem {
```
Comment (JoakimLofgren) [inline Showroom/Sidebar/styles.scss]: Remove?

File: Showroom/states.js [hunk 12]
```diff
@@ -1,45 +1,67 @@
...
+        grid: reduceResize(state.grid, action.payload)
       }
 
-    case 'TOGGLE_GRID':
-      return {
-        ...state,
-        grid: {
-          ...state.grid,
-          display: !state.grid.display
+    case 'ROUTE':
```
Comment (JoakimLofgren) [inline Showroom/states.js]: Using consts in both the action and the reducer helps prevent typos :)

Comment (xaviervia) [inline Showroom/states.js]: Done in 6b0587f

File: Code/index.js [hunk 13]
```diff
@@ -0,0 +1,2 @@
+export Block from './Block'
```
Comment (xaviervia) [inline Code/index.js]: This is actually related to having a built version: with a built version, this is a non-issue. Let's follow up on https://github.com/klarna/ui/issues/81

File: Showroom/Footer/index.jsx (hunk 5 above)
Comment (xaviervia) [inline Showroom/Footer/index.jsx]: So, will follow up in https://github.com/klarna/ui/issues/84

**Pull request merged**